decidim-proposals/config/locales/en.yml

```diff
@@ -410,8 +414,6 @@ en:
           no_similars_found: Well done! No similar collaborative drafts found
           title: Similar collaborative drafts
         complete:
-          attachment_legend: "(Optional) Add an attachment"
-          select_a_category: Please select a category
```

Comment:
Why this?

Reply:
It is explained in [this commit](https://github.com/decidim/decidim/pull/4585/commits/1216a62d3a3d1185cee617e9a37637963122b8ae): I've refactored collaborative drafts templates using a partial, so I reduced a bit the need of translating the same text several times. Also, I changed some translation keys paths included in that partial, that is way I touched all the locales files, to reuse existing translation instead of translating it again.